Makefile

```diff
@@ -563,6 +563,61 @@ kind-image: ## Build cilium and operator images and import them into kind.
 	$(MAKE) kind-image-agent
 	$(MAKE) kind-image-operator
 
+.PHONY: kind-install-cilium-fast
+kind-install-cilium-fast: kind-ready ## Install a local Cilium version into the cluster.
```

Comment:
We could maybe refactor this with `kind-install-cilium` by making a function and pass the Helm value file to use as argument. Although, I'm also fine with the current version.

Reply:
Yes, I thought about it but it sounded more messy.